EventsExpress/ClientApp/src/security/authComponent.js

```diff
@@ -3,15 +3,10 @@ import { connect } from 'react-redux';
 
 class AuthComponent extends Component {
     render() {
-        if(this.props.roleMatch){
-            if (this.props.id && this.props.role == this.props.roleMatch) {
-                return this.props.children;
-            }
-        }
-        else{
-            if (this.props.id) {
-                return this.props.children;
-            }
+        const {id, roles, roleMatch, children} = this.props;
+
+        if( id && (roleMatch ? roles.includes(roleMatch) : true )){
```

Comment:
Why do we use `includes` here and `some` in the othre component? Does it make sense to use `some` here as well?